src/Microsoft.TestPlatform.CommunicationUtilities/Microsoft.TestPlatform.CommunicationUtilities.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>8.0.3</Version>
+      <Version>9.0.1</Version>
```

Comment:
I believe communication utilities is loaded inside VS IDE, which takes a dependency on 8.0.3, having 9.0.1 will break VS functionalities,
Adding Arun @codito for confirmation.